var/spack/repos/builtin/packages/py-mdanalysistests/package.py

```diff
@@ -14,16 +14,20 @@ class PyMdanalysistests(PythonPackage):
 
     maintainers("RMeli")
 
+    version("2.6.1", sha256="043f7451f4d9c42ea9e6609a81a6002948e2c74fd268282e0831416789b22e5e")
+    version("2.6.0", sha256="16fdd10e5240b606e8f9210b7cbd9e4be110e6b8d79bb6e72ce6250c4731a817")
     version("2.5.0", sha256="a15b53b7f8bed67900a2bf542bbb3cab81dc71674fa6cddb3248dd11880e4c9d")
     version("2.4.3", sha256="6fbdeccdbfb249f76520ee3605d007cd70292187e3754d0184c71e5afe133abb")
     version("2.4.2", sha256="6e8fb210a4268691c77717ea5157e82d85874a4f7ee0f8f177718451a44ee793")
 
     # Version need to match MDAnalysis'
```

Comment:
Could use a for-loop here if you want

Reply:
Good point. Is there an easy way to get a list of all the defined versions?

Reply:
Not that I know of unfortunately

Reply:
Thanks. In such case I'm not sure I really see the benefit of a for loop. Versions still need to manually defined, but it will obfuscate the package definition IMO.